python/setup.py

```diff
@@ -1,14 +1,16 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
-    name='FogLAMP',
+    python_requires='==3.5.3',
+    name='foglamp',
     version='0.1',
     description='FogLAMP',
     url='http://github.com/foglamp/FogLAMP',
     author='OSIsoft, LLC',
     author_email='info@dianomic.com',
     license='Apache 2.0',
-    packages=['foglamp'],
+    # TODO: list of excludes (tests)
+    packages=find_packages(),
```

Comment:
the __init__.py needed to be added to the new dir dtructure for find_packages to work